apps/services/user-profile/src/app/v2/dto/user-profile.dto.ts

```diff
@@ -53,4 +54,14 @@ export class UserProfileDto {
   @IsOptional()
   @IsBoolean()
   readonly needsNudge?: boolean | null
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
+  readonly emailStatus?: DataStatus
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
+  readonly mobileStatus?: DataStatus
```

Comment:
We should not put those fields in the API response.

Reply:
Nice catch, forgot to remove it :D